### PR TITLE
Add a small doc detail for BackdropFilter

### PR DESCRIPTION
**Pull request by xster**
#10148

### CODE REVIEW

File: packages/flutter/lib/src/widgets/basic.dart [hunk 1]
```diff
@@ -149,6 +149,21 @@ class ShaderMask extends SingleChildRenderObjectWidget {
 ///
 /// This effect is relatively expensive, especially if the filter is non-local,
 /// such as a blur.
+///
+/// The effect is applied to all existing painted content. To constrain
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/basic.dart]: all existing content? not just of the size of this render object? that seems surprising.

Comment (xster) [inline packages/flutter/lib/src/widgets/basic.dart]: @collinjackson that's what I seem to have understood from our conversation. Or I might have gotten it wrong.

Comment (Hixie) [inline packages/flutter/lib/src/widgets/basic.dart]: that seems like a bug, if so. I would have expected it to create a new layer, the size of the render object, then apply the effect to everything in the part of the parent layer that is directly under that layer.

Comment (collinjackson) [inline packages/flutter/lib/src/widgets/basic.dart]: I agree that the behavior is surprising and I probably should have filed a bug on it at the time.

Comment (Hixie) [inline packages/flutter/lib/src/widgets/basic.dart]: Yeah let's file a bug before we just document this surprising behaviour. If Skia says it's what they intended, we can add the docs then.

**Comment by Hixie** [comment]
@collinjackson since you understand this best could you file a bug for it then close this PR? Thanks!

**Comment by collinjackson** [comment]
Will do! Thanks

**Comment by Hixie** [comment]
@collinjackson Did you ever have a chance to file that bug? (I'm just going through cleaning up old PRs.)

**Comment by collinjackson** [comment]
Here is a [test case app](https://gist.github.com/collinjackson/4318255c6390f080f91011329a068d62).

The following are the bugs that I will file on Skia.

1. BackdropFilter bleeds outside its bounds when text is drawn on it. It expands to cover all the layers below it in a Stack (which can expand out to include the entire screen) if it's not contained by a `ClipRect`.

webp net-gifmaker 2

2. BackdropFilter does nothing if the contents are clipped with ClipOval. (I think ClipRRect has this problem too, but ClipRect is fine.)

inner oval

3. BackdropFilter does nothing if it's contained in a ClipOval. (I think ClipRRect has this problem too, but ClipRect is fine.)

webp net-gifmaker 1

**Comment by collinjackson** [comment]
This seems like a bug somewhere, either in Flutter or Skia, so let's close this.

**Comment by collinjackson** [comment]
Here's the [Skia bug I filed](https://bugs.chromium.org/p/skia/issues/detail?id=6753)

**Comment by xster** [comment]
Awesome! Thanks for testing. 

I can confirm seeing all 3 issues when I was testing initially (with ClipRRect). I was definitely surprised by the text behaviour too.

**Pull request closed without merging**